astropy/coordinates/tests/test_unit_representation.py

```diff
@@ -45,18 +45,20 @@ def __new__(cls, angle, unit=None, wrap_angle=180*u.deg, **kwargs):
                                    **kwargs)
             return self
 
-    class UnitSphericalWrap180Representation(UnitSphericalRepresentation):
-        attr_classes = OrderedDict([('lon', Longitude180),
-                                    ('lat', Latitude)])
-        recommended_units = {'lon': u.deg, 'lat': u.deg}
+    with pytest.warns(AstropyDeprecationWarning,
+                      match="The 'recommended_units' attribute is deprecated"):
+        class UnitSphericalWrap180Representation(UnitSphericalRepresentation):
+            attr_classes = OrderedDict([('lon', Longitude180),
+                                        ('lat', Latitude)])
+            recommended_units = {'lon': u.deg, 'lat': u.deg}
```

Comment:
Similarly, these can be removed as well.

Reply:
p.s. Sorry that I didn't clean that up better! Clearly, it is useful to have pytest fail if there are unexpected warnings!